components/textfield/stories/template.js

```diff
@@ -183,6 +188,98 @@ export const Template = ({
 			size: "s",
 			customClasses: customProgressCircleClasses,
 		}, context))}
+		${when(helpText, () =>
+				HelpText({
+					text: helpText,
+					variant: isInvalid ? "negative" : "neutral",
+					size,
+					hideIcon: true,
+					isDisabled
+				}, context ))}
 	</div>
 	`;
 };
+
+export const HelpTextOptions = (args, context) => Container({
+	direction: "column",
+	withBorder: false,
+	withHeading: false,
+	content: html`
+		${Container({
+			withBorder: false,
+			heading: "Description",
+			content: Template({...args, isRequired: true, labelText: "Username", value: "lisawilson24", helpText: "Username must be at least 8 characters."}, context),
+		})}
+		${Container({
+			withBorder: false,
+			heading: "Error message",
+			content: Template({...args, isRequired: true, labelText: "Email address", value: "abc@adobe.com", helpText: "Enter your email address", isInvalid: true }, context),
+		})}
+	`
+});
+
+export const QuietGroup = (args, context) => Container({
+	direction: "row",
+	withBorder: false,
+	wrapperStyles: {
+		rowGap: "12px",
+	},
+	content: html`
+		${Container({
+			withBorder: false,
+			containerStyles: {
+				"gap": "8px",
+			},
+			heading: "Default",
+			content: Template({args, context, isQuiet: true})
+		})}
+		${Container({
+			withBorder: false,
+			containerStyles: {
+				"gap": "8px",
+			},
+			heading: "Invalid",
+			content: Template({...args, isInvalid: true, isQuiet: true}, context)
+		})}
+		${Container({
+			withBorder: false,
+			containerStyles: {
+				"gap": "8px",
+			},
+			heading: "Hovered",
```

Comment:
I don't think we have an `isHovered` arg. Is this "Hovered" case removable, or are we missing that arg? (I think it's missing, since we do have styles for hover)